packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -400,6 +400,30 @@ pool.addClient('app-client', {
 });
 ```
 
+Clients can be configured so that only specific user attributes can be read or modified by it.
+The default behavior is to allow read and write all available attributes, but it is often not the best option.
+For example, consider a webapp that contains a custom attribute `customerPlan` to store the user's pricing plan
+where only some clients, such as the administrators, should be allowed to change this value. Read more about [Attribute
+Permissions and Scopes](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html#user-pool-settings-attribute-permissions-and-scopes).
+
+```ts
+const pool = new UserPool(this, 'Pool');
+pool.addClient('webapp-client', {
+  readAttributes: {
+    standard: [StandardAttribute.NICKNAME, StandardAttribute.EMAIL, StandardAttribute.NAME],
+    custom: ['customerPlan'],
+  },
+  writeAttributes: {
+    standard: [StandardAttribute.NICKNAME],
+  }, // only the nickname will be writable
+});
+```
+
+The default behavior of allowing read/write of all attributes is enforced at Client creation, and only those
+attributes available at that time will be included.
+If any attribute is added to the User Pool after the creation of the Client, that attribute will not be
+available unless explicitly specified.
+
```

Comment:
I've added those lines but I'm not sure it's easy to understand what i mean. @nija-at could you please give me a feedback?

Reply:
This applies only for custom attributes, no? Because I thought standard attributes are always available.